hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1072,6 +1074,19 @@ message KeyInfo {
     optional bool isFile = 19;
 }
 
+message BasicKeyInfo {
+    required string volumeName = 1;
+    required string bucketName = 2;
+    required string keyName = 3;
+    required uint64 dataSize = 4;
+    required uint64 creationTime = 5;
+    required uint64 modificationTime = 6;
+    required hadoop.hdds.ReplicationType type = 7;
+    optional hadoop.hdds.ReplicationFactor factor = 8;
+    optional hadoop.hdds.ECReplicationConfig ecReplicationConfig = 9;
```

Comment:
S3 APIs don't need this, and we can skip it. In most cases, this will be the same as the setting for the bucket, in which case the client can discover it from the bucket info.

Reply:
Thank you for the suggestion! I will address this along with this [comment](https://github.com/apache/ozone/pull/5115#discussion_r1298020212).